core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -39,19 +39,28 @@
                       !canWindowShowTwoCards() && !isCurrentSupplementalCardNonempty()}">
       <tutor-card on-submit-answer="submitAnswer(answer, rulesService)"
                   is-learn-again-button="isLearnAgainButton()"
+                  ng-if="!conceptCardShown"
+                  concept-card-pending="conceptCardPending"
                   start-card-change-animation="startCardChangeAnimation"
                   on-change-interaction-answer-validity="setInteractionAnswerValidity(answerValidity)">
       </tutor-card>
+      <concept-card skill-id="missingPrerequisiteSkillId"
+                    ng-if="conceptCardShown"
+                    return-to-exploration="returnToExploration()">
+      </concept-card>
       <div ng-if="isCurrentSupplementalCardNonempty() && !canWindowShowTwoCards()">
         <supplemental-card on-click-continue-button="showUpcomingCard()"
                            is-learn-again-button="isLearnAgainButton()"
+                           ng-if="!conceptCardShown"
```

Comment:
Perhaps this should be part of the ng-if in the parent.

Reply:
Done.